src/mumble/VoiceRecorder.cpp

```diff
@@ -277,6 +289,16 @@ bool VoiceRecorder::ensureFileIsOpenedFor(SF_INFO &soundFileInfo, boost::shared_
 		return false;
 	}
 
+#ifdef USE_SNDFILE_MP3
+	// Enable constant 320kbps for MP3 formats
+	if (m_config.recordingFormat == VoiceRecorderFormat::MP3) {
+		int bitrate        = SF_BITRATE_MODE_CONSTANT;
+		double compression = 0.0;
+		sf_command(ri->soundFile, SFC_SET_BITRATE_MODE, &bitrate, sizeof(int));
+		sf_command(ri->soundFile, SFC_SET_COMPRESSION_LEVEL, &compression, sizeof(double));
```

Comment:
I'm wondering whether these changes might have an effect on subsequent recordings where the chosen format is something other than MP3? Have you tested that?

Reply:
Should I just get rid of it then? I think you're right about it having effects on future recordings.

Reply:
I don't know. You probably added them for a reason, no? If they don't do anything, then remove them, but if they are important/useful for MP3 recordings, they should be kept and some mechanism of restoring the previous settings when using other recording formats should be implemented :thinking: